lib/identity/client.js

```diff
@@ -328,6 +328,24 @@ class Client extends PartialClient {
     )
     return await validateResponseAsJSON(response)
   }
+
+  /**
+   * privateRealmInfo fetches private information about a realm
+   *
+   * @return {Object} information about a realm
+   */
+  async privateRealmInfo() {
+    let response = await this.storage.authenticator.tsv1Fetch(
```

Comment:
It's worth considering adding a cache for this info... If you look at how the basic `realmInfo()` method works, there is an in memory cache that is checked first and returned if the data has already been fetched once. There is also an invalidation flag allowing the user to forcibly fetch fresh data from the server.

See: https://github.com/tozny/js-sdk/blob/3c6d548fc4345569328d01752ab9f1812c43304f/lib/identity/realm.js#L149

For this, the cache should be scoped to the identity object, not globally as the public info cache is (that one uses an array stored globally in the module, this would be better stored as an internal object property).